packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3476,7 +3476,6 @@ export class ContainerRuntime
 			await delay(this.closeSummarizerDelayMs);
 			this._summarizer?.stop("latestSummaryStateStale");
 			this.closeFn();
-			throw error;
```

Comment:
#nit if we no longer need the error, we can just drop the cost and create the error on the fly in the telemetry event.